src/test/java/org/alfresco/repo/web/scripts/servlet/RemoteAuthenticatorFactoryAdminConsoleAccessTest.java

```diff
@@ -198,6 +212,35 @@ public void testAdminCanAccessAdminConsoleHelperScript()
         complexCheckOfScriptCases(families);
     }
 
+    /**
+     * Tested access to the AdminConsole for an non literal admin user
+     * but with admin permissions (user added to ALFRESCO_ADMINISTRATORS)
+     * and accessing via Basic Auth
+     */
+    @Test
+    public void testUserCanAccessAdminConsoleScript()
+    {
+        Set<String> families = new HashSet<>();
+        families.add("AdminConsole");
+
+        // Run as System to be able to give permissions
+        this.authenticationComponent.setSystemUserAsCurrentUser();
+
+        String username = RandomStringUtils.randomAlphabetic(10);
+        String password = RandomStringUtils.randomAlphabetic(10);
+        this.createUser(username, password);
```

Comment:
What is the reason for adding "this" keyword?

Reply:
I don't think any of "this" are required:
https://docs.oracle.com/javase/tutorial/java/javaOO/thiskey.html